include/swift/SIL/FieldSensitivePrunedLiveness.h

```diff
@@ -345,6 +345,16 @@ struct TypeTreeLeafTypeRange {
     return TypeTreeLeafTypeRange(start, end);
   }
 
+  /// Whether \p bits contains any of the in-range bits.
+  bool intersects(SmallBitVector const &bits) const {
+    for (auto bit : bits.set_bits()) {
```

Comment:
Doesn't it make more sense to iterate over the range of elements (for this particular instruction) rather then over all the bits in the type?

Reply:
Fixed.